v3/docs/TUTORIAL.md

```diff
@@ -347,14 +347,14 @@ asyncio.run(main())
  * `asyncio.run` Arg: the coro to run. Return value: any value returned by the
  passed coro. The scheduler queues the passed coro to run ASAP. The coro arg is
  specified with function call syntax with any required arguments passed. In the
- current version the `run` call returns when the task terminates. However under
- CPython the `run` call does not terminate.
+ current version the `run` call returns when the task terminates. However, under
+ CPython, the `run` call does not terminate.
  * `await`  Arg: the task or coro to run. If a coro is passed it must be
  specified with function call syntax. Starts the task ASAP. The awaiting task
  blocks until the awaited one has run to completion. As described
- [in section 2.2](./TUTORIAL.md#22-coroutines-and-tasks) it is possible to
- `await` a task which has already been started. In this instance the `await` is
- on the `task` object (function call syntax is not used).
+ [in section 2.2](./TUTORIAL.md#22-coroutines-and-tasks), it is possible to
```

Comment:
I'm not sure this is easier to read with its nested parentheses.

Reply:
Reverted